build_msvc/bitcoin_config.h.in

```diff
@@ -29,6 +29,9 @@
 /* Copyright year */
 #define COPYRIGHT_YEAR $
 
+/* Define this symbol to build in assembly routines */
+#define USE_ASM
```

Comment:
NIT: From what I understand we are enabling the special CPU instructions using intrinsics and not inline assembly. This makes the comment a little misleading and makes the symbol unfortunately named.

Perhaps the symbol could be?

```suggestion
/* Define this symbol to enable CPU SHA Extensions */
#define USE_SHA_EXTENTIONS
```

or?

```suggestion
/* Define this symbol to enable CPU Extensions */
#define USE_INTRINSICS
```